spec/coffee/directives/api/utils/_async.spec.coffee

```diff
@@ -2,21 +2,23 @@ describe "_async", ->
   rootScope = null
   timeout = null
   q = null
+  browser = null
 
   digest = (fn, times = 1) =>
     fn()
     if times
       _.range(times).forEach -> # i would like to say that it sucks that I have to do this.. (angular)
-        timeout?.flush()
+        timeout?.flush() if browser.deferredFns?.length
```

Comment:
Unless this is meant through erros and in that case I trap it and make a bool out of it for the while loops. This really seems kinda dumb, as it should be a bool to begin with.